src/test/rustdoc/copy-local-img.rs

```diff
@@ -0,0 +1,10 @@
+#![crate_name = "foo"]
+
+// @has static/8a40d4987fbb905
+// @has foo/struct.Enum.html
+// @has - '//img[@src="../static/8a40d4987fbb905"]' ''
```

Comment:
I fear these hashes won't be consistent between systems now that the filename is canonicalised.  Is there a cut-down test which will still work here?

Reply:
Oh, didn't think about that, the hash will fail indeed. Great catch! I can add something to count files in the given folder and add a "start with" thing. Unless you have another idea?

Reply:
One option could indeed be that.  Another -- hash the file as part of the deduplication effort, but then simply number them as you add them to the `static/` dir?